doc/interop-test-descriptions.md

```diff
@@ -452,10 +466,9 @@ Asserts:
 
 ### unimplemented_method
 
-Status: Not yet implementable
+Status: Ready for implementation. Blocking beta.
```

Comment:
Agreed. Will do that as a separate PR since it will need to trigger all testing.